internal/lang/functions.go

```diff
@@ -169,32 +165,3 @@ func (s *Scope) Functions() map[string]function.Function {
 
 	return s.funcs
 }
-
-// experimentalFunction checks whether the given experiment is enabled for
-// the recieving scope. If so, it will return the given function verbatim.
-// If not, it will return a placeholder function that just returns an
-// error explaining that the function requires the experiment to be enabled.
-func (s *Scope) experimentalFunction(experiment experiments.Experiment, fn function.Function) function.Function {
```

Comment:
I guess this is probably now getting caught by our recently-added unused function lint... but is there some way we can tell it to let us keep this here? This is a non-trivial method that we'll probably need to reuse at some later point for another experiment, so I'd rather avoid digging around in the Git history to try to find it again next time we have another experimental function. :grimacing:

Reply:
Yep, I only removed it because staticcheck yelled at me.

I've added a linter directive to disable the unused function error and re-added the function. We'll have to remove that directive if the function is used again but that's easier than Git spelunking.